vm/mock/systemEIStub.go

```diff
@@ -1,8 +1,10 @@
 package mock
 
 import (
-	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/process/smartContract/hooks"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
```

Comment:
vmcommon prefix is not needed.

Reply:
Go tools from vscode force me to do this.